samples/VirtoCommerce.NotificationsSampleModule.Web/NotificationLayoutsTemplate/SampleNotificationLayout.html

```diff
@@ -0,0 +1 @@
+<p>Sample Text</p>
```

Comment:
Add placeholder for content
```
<div>Sample Header</div>
     {{ content }}
<div>Sample Footer</div>
```

https://docs.virtocommerce.org/modules/notification/#working-with-layouts